### DISCUSSION TITLE
Releases Update (10.7.0 as latest)

### DISCUSSION THREAD

**Discussion by CustomerJFK** [jellyfin/jellyfin#4941] [General] [answered]
Hello
When will be the Release 10.7.0 available as a official (latest) Release for Dockers ????

BR

**Comment by nielsvanvelzen** [comment, marked as answer]
@joshuaboniface replied to a similar question on Reddit:

> Assuming that we don't find any major bugs in rc2, we'll give it a week or two then do the real release to be sure. If we do find a major bug that needs an extensive fix, there may be an rc3 before the real release.

https://www.reddit.com/r/jellyfin/comments/ko1s7r/jellyfin_1070_release_candidate_2_incoming_now/ghnuotn
